en/advanced_config/advanced_mc_position_tuning.md

```diff
@@ -1,96 +1,103 @@
-<!-- Fixes page load bug in notation, courtesy https://github.com/GitbookIO/plugin-mathjax/issues/34#issuecomment-349453673 -->
-<script>
-gitbook.events.bind("page.change", function() {
-    MathJax.Hub.Queue(["Typeset",MathJax.Hub]);
-}
-</script>
-
 # Advanced Multicopter Position Control Tuning
- The objective of this document is to give an overview of the higher-level position-control tuning parameters for multicopters. Higher-level tuning applies to any parameter that has an effect on
- the desired position/velocity setpoint as opposed to [lower-level tuning](pid_tuning_guide_multicopter.md) that influences the tracking.
- The set of higher-level tuning parameters can be split into two sections: tunig parameters for manual control position mode and tuning parameters for auto. Some parameters will have an effect on both
- modes. 
 
-> **Warning** This guide is for advanced users/experts. 
+This document provides an overview of the higher-level position-control tuning parameters for multicopters.
+
+> **Warning** This guide is for advanced users/experts.
+
+<span></span>
+> **Tip** Follow the instructions in the [Multicopter PID Tuning Guide](https://docs.px4.io/en/advanced_config/pid_tuning_guide_multicopter.html) *before* doing any of the higher-level related control tuning described here. Do not use the advanced position control tuning parameters to fix bad tracking or vibration!
+
+## Overview
 
-## Short Background
-The position controller consists of an outer P-position control loop and an inner PID-velocity control loop. Depending on the control mode either
-the velocity control loop only is active or both. For the remaining part of this tutorial the term **position-control** represents the position and velocity control loop
-activated and **velocity-control** when just the velocity control loop is in use.
+Higher-level tuning applies to any parameter that has an effect on the desired position/velocity setpoint, in contrast to [lower-level tuning](pid_tuning_guide_multicopter.md) that influences the tracking.
 
-## Precondition
-Before doing any higher-level related control tuning, please follow first the instruction [PID-tuning](https://docs.px4.io/en/advanced_config/pid_tuning_guide_multicopter.html).
-It is not recommended to use the advanced position control tuning parameters as a fix for bad tracking or vibration.
+The set of higher-level tuning parameters can be split into two sections: tuning parameters for [Manual/Stabilized](#manual_mode) mode and tuning parameters for [Mission](#mission_mode) mode. Some parameters will have an effect on both modes. 
```

Comment:
@Stifael 
1. FYI Auto mode is now called "Mission". In the user guide we always refer to the modes by the name that they have in QGC selection.
2. Similarly, I THINK that when you refer to manual position control mode you mean "Manual/Stabilized" mode.

Reply:
1. Ok
2. No. Manual/Stabilized is the mode where you have no position/velocity control active. The user "controls" the attitude and thrust via RC stick inputs. What I am referring to is Manual for RC stick inputs together with position/velocity control active. In QGC it is called Position.

Reply:
OK, so I assume this would be right if I just change to heading/link to Position mode.